openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -513,12 +512,7 @@ def visit_sequential(self, course, chapter, sequential):
         )
         self.assertEqual(200, self.client.get(last_accessed_url).status_code)
 
-    @override_switch(
-        '{}.{}'.format(
-            waffle.WAFFLE_NAMESPACE, waffle.ENABLE_COMPLETION_TRACKING
-        ),
-        active=True
-    )
+    @override_waffle_switch(waffle.ENABLE_COMPLETION_TRACKING_SWITCH, active=True)
```

Comment:
Also search for `waffle.ENABLE_COMPLETION_TRACKING_SWITCH` and fix imports. Thanks.